src/index.ts

```diff
@@ -101,6 +102,7 @@ function getOptions(
         stopDir: os.homedir(),
         searchPlaces: getDefaultSearchPlaces(name),
         ignoreEmptySearchPlaces: true,
+        cache: true,
```

Comment:
Note: Setting by default `cache` to `true` is certainly a breaking change for downstream consumers. If you want to keep it this way, I would recommend releasing this change under a new major version.

Reply:
>  I consider this change in behaviour a breaking change.

I haven't looked at the PR description. 😂